### PR TITLE
Nextjs UI template guide

### PR DESCRIPTION
**Pull request by tonyboylehub**

### CODE REVIEW

File: src/components/products/guides/index.js [hunk 1]
```diff
@@ -116,6 +116,17 @@ export const guides = {
             },
           ],
         },
+        {
+          title: 'Templates',
+          links: [
+            {
+              title: 'NextJs Template',
+              href: '/guides/templates/nextjs',
```
Comment (MarkSackerberg) [inline src/components/products/guides/index.js:124]: seo wise: should we add solana, umi or something like that into the uri?

File: src/pages/guides/templates/nextjs.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+// Pulls umi state from the umiStore.
+const umi = useUmiStore.getState().umi
+const signer = useUmiStore.getState().signer
+
+umi.use(signerIdentity(signer))
+```
+
+## Helpers
+
+Stored in the `/lib/umi` folder there are some pre made helps you can use to make your development easier.
```
Comment (MarkSackerberg) [inline src/pages/guides/templates/nextjs.md:97]:
```suggestion
Stored in the `/lib/umi` folder there are some pre made helpers you can use to make your development easier.
```

File: src/pages/guides/templates/nextjs.md [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+
+Umi is split up into several components which can be called in different scenarios.
+
+#### sendAndConfirmWithWalletAdapter()
+
+Passing a transaction into `sendAndConfirmWithWalletAdapter()` will send the transaction while pulling the latest walletAdapter state from the zustand `umiStore` and will return the signature as a `string`. This can be accessed in both `.ts` and `.tsx` files.
+
+The function also provides and locks in the commitment level across `blockhash`, `send`, and `confirm` if provide. By default `confirmed` is used.
```
Comment (MarkSackerberg) [inline src/pages/guides/templates/nextjs.md:105]:
```suggestion
The function also provides and locks in the commitment level across `blockhash`, `send`, and `confirm` if provided. By default `confirmed` is used.
```

**Review by MarkSackerberg — APPROVED** [review]
some minor nitpicks.
Thank you for implementing this!

File: src/components/products/guides/index.js (hunk 1 above)
Comment (tonyboylehub) [reply src/components/products/guides/index.js:124]: yeah I came up with a naming convention for templates a number of weeks ago

`metaplex-reactframework-componentLib-cssframework-template`

Examples
`metaplex-nextjs-tailwind-template`
`metaplex-nextjs-shadcn-tailwind-template`
`metaplex-nextjs-manetime-template`
`metaplex-react-manetime-template`
`metaplex-react-tailwind-template`

I should probably make the URI and title based on this.

**Review by MarkSackerberg — DISMISSED** [review]
missed the new review request, sorry!

**Review by MarkSackerberg — APPROVED** [review]

**Pull request merged**